readthedocs/oauth/services/bitbucket.py

```diff
@@ -393,3 +394,7 @@ def update_webhook(self, project, integration):
             log.exception("Bitbucket webhook update failed for project.")
 
         return (False, resp)
+
+    def send_build_status(self, build, commit, status):
+        """Send build status is not supported/implemented for Bitbucket."""
+        return True
```

Comment:
Seems odd we're returning `True` when it's not supported?

Reply:
Our current logic expects at least one provider to return True, otherwise it will keep trying, and show an error to the user if none of those returned True. Another way could be to have an attribute like `supports_build_status`.

Reply:
> Another way could be to have an attribute like supports_build_status.

Yea, that seems like a nicer solution?